pkg/apis/options/load.go

```diff
@@ -132,3 +135,28 @@ func isUnexported(name string) bool {
 	first := string(name[0])
 	return first == strings.ToLower(first)
 }
+
+// LoadYAML will load a YAML based configuration file into the options interface provided.
+func LoadYAML(configFileName string, into interface{}) error {
+	v := viper.New()
+	v.SetConfigFile(configFileName)
+	v.SetConfigType("yaml")
+	v.SetTypeByDefaultValue(true)
+
+	if configFileName == "" {
+		return errors.New("no configuration file provided")
+	}
+
+	data, err := ioutil.ReadFile(configFileName)
+	if err != nil {
+		return fmt.Errorf("unable to load config file: %w", err)
+	}
+
+	// UnmarhsalExact will return an error if the config includes options that are
```

Comment:
```suggestion
	// UnmarshalExact will return an error if the config includes options that are
```

Reply:
Should actually be `UnmarshalStrict` now, changed yaml library and forgot to update the comment 😅